modules/tinymce/src/core/main/ts/fmt/ExpandRange.ts

```diff
@@ -20,12 +20,20 @@ import * as FormatUtils from './FormatUtils';
 
 type Sibling = 'previousSibling' | 'nextSibling';
 
+interface DataNode {
+  data: string;
+}
+
+const isDataNode = (node: Node | DataNode): node is DataNode => node.hasOwnProperty('data');
```

Comment:
Isn't it the Text node that you're interested in (it has the data property) in which case you can use NodeType.isText?

Reply:
Ah, so that was the type it was.